Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'pg'
 gem 'rails', rails_version
 
 group :development do
-  gem 'rspec-rails'
+  gem 'rspec-rails', "~> 3.1.0"
```

Comment:
Somebody already updated that in the gemspec without also updating the Gemfile. Regenerating the gemfile caused that constraint to be lost.